src/psyclone/tests/psyir/nodes/acc_directives_test.py

```diff
@@ -68,41 +68,28 @@ def setup():
     Config._instance = None
 
 
```

Comment:
As per our discussion in the meeting on Monday, please could you precede this test with a comment saying which class method it is for (see later in this file for examples). In fact, we also need a comment to say which class the following tests are for (i.e. ACCLoopDirective). Please could you ensure that this test is in the right location in this file - i.e. that it follows the ordering of the classes in the `acc_directives.py` file.

Reply:
Done. I reordered all test classes that weren't on the right location. This certainly exposes where we have coverage but we are not really unit testing as some classes are even missing from this file.

Reply:
Thanks Sergi.